components/Header/index.js

```diff
@@ -0,0 +1,19 @@
+import styled from "styled-components";
+
+export default function Header() {
+  return <StyledHeader>PlantBuddy</StyledHeader>;
+}
```

Comment:
As your `h1` element has no logic you only need the styled component and the react component is not needed in this case. I would name the styled component also `StyledHeadline` and not `StyledHeader` as you would usually think that this is a styled `header` element and not a `h1`. :)

As you use your Header for multiple pages you can think about to implement a Layout component that you add to your `_app.js` that has all the necessary components and stylings that each page should have. :) See as example [this Layout component for an example Art Gallery App](https://github.com/mbosselmann/art-gallery-app/blob/main/components/Layout.js) and [how it is integrated in the _app.js](https://github.com/mbosselmann/art-gallery-app/blob/main/pages/_app.js).